libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreInspectorViewController.m

```diff
@@ -88,14 +88,28 @@ + (SFSmartStoreInspectorViewController *) sharedInstance
 
 #pragma mark - Present / dimiss
 
-+ (void) present
++ (void) present:(UIViewController*)currentViewController
 {
-    [[SFRootViewManager sharedManager] pushViewController:[SFSmartStoreInspectorViewController sharedInstance]];
+    if (![NSThread isMainThread]) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            [SFSmartStoreInspectorViewController present:currentViewController];
+        });
+        return;
+    }
+
+    [currentViewController presentViewController:[SFSmartStoreInspectorViewController sharedInstance] animated:NO completion:nil];
```

Comment:
Ah, that makes sense.  As far as the `present` and `dismiss` methods go, it seems like we're writing convenience wrappers around the one-liners `presentViewController:animated:completion` and `dismissViewControllerAnimated:completion`.  Are we adding value there to developers?